tests/workflows/test_uber_lyft.py

```diff
@@ -0,0 +1,67 @@
+import uuid
+
+import coiled
+import dask.dataframe as dd
+import pytest
+from dask.distributed import Client
+
+
+@pytest.fixture(scope="module")
+def uber_lyft_cluster(
+    dask_env_variables,
+    cluster_kwargs,
+    github_cluster_tags,
+):
+    with coiled.Cluster(
+        f"uber-lyft-{uuid.uuid4().hex[:8]}",
+        environ=dask_env_variables,
+        tags=github_cluster_tags,
+        **cluster_kwargs["uber_lyft_cluster"],
+    ) as cluster:
+        yield cluster
+
+
+@pytest.fixture
+def uber_lyft_client(
+    uber_lyft_cluster,
+    cluster_kwargs,
+    upload_cluster_dump,
+    benchmark_all,
+):
+    n_workers = cluster_kwargs["uber_lyft_cluster"]["n_workers"]
+    with Client(uber_lyft_cluster) as client:
+        uber_lyft_cluster.scale(n_workers)
+        client.wait_for_workers(n_workers)
+        client.restart()
+        with upload_cluster_dump(client), benchmark_all(client):
+            yield client
+
+
+@pytest.fixture
+def ddf(uber_lyft_client):
+    """NYC taxi Uber/Lyft dataset"""
+    paths = [
+        f"s3://coiled-datasets/mrocklin/nyc-taxi-fhv/{year}-*.parquet"
+        for year in range(2019, 2023)
+    ]
+    return dd.read_parquet(paths, engine="pyarrow", storage_options={"anon": True})
```

Comment:
We have a fixture to read parquet using string pyarrows, See https://github.com/coiled/coiled-runtime/blob/decbba56de7158b590b1e79fd7b6eb490838df09/tests/conftest.py#L674-L698

and this is how you can use it https://github.com/coiled/coiled-runtime/blob/8f159e9d0e1ea97874fd751fce3607c3f025ff0a/tests/benchmarks/test_xgboost.py#L33